packages/oas/medusa-oas-cli/README.md

```diff
@@ -0,0 +1,60 @@
+# medusa-oas-cli - 0.1.0 - experimental
+
+A command-line tool for all OAS related tooling.
+
+## Install
+
+`yarn add --dev @medusajs/medusa-oas-cli`
+
+Install in the global namespace is not yet supported.
+~~`npm install -g @medusajs/medusa-oas-cli`~~
+
+## Configuration / First time setup
+
+N/A
+
+## How to use
+
+`yarn medusa-oas <command>`
+
+### Command - `oas`
+
+Will scan the `@medusajs/medusa` package in order to extract JSDoc OAS into a json file.
+
+Be default, the command will output two files, `admin.oas.json` and `store.oas.json`, in the same directory that
+the command was run.
```

Comment:
Should be on the same line as the previous sentence.

Reply:
It is put on a separate line automatically by my IDE since the sentence is longer than 120 characters.